### PR TITLE
DTS2159378-Fix AggregateErrorPart5

### PR DESCRIPTION
**Pull request by garun-kumar**
**Task name**: PackerBuildV1,JenkinsDownloadArtifactsV1,KubeloginInstallerV0,

**Description**:  DTS 2159378

**Documentation changes required:** /

**Added unit tests:** N

**Attached related issue:**[DTS TASK 2159378](https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/2159378)

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/PackerBuildV1/Tests/L0CustomManagedTemplate.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ tr.setInput("additionalBuilderParameters", "{}");
 tr.setInput("customTemplateParameters", "{\"client_id\": \"abcdef\", \"drop-location\":\"C:\\\\folder 1\\\\folder-2\"}");
 
 process.env["RELEASE_RELEASENAME"] = "Release-1";
-
+process.env['AGENT_TEMPDIRECTORY'] = '.';
```
Comment (z00sts) [inline Tasks/PackerBuildV1/Tests/L0CustomManagedTemplate.ts:16]: Can you explain adding this change?

Comment (garun-kumar) [reply Tasks/PackerBuildV1/Tests/L0CustomManagedTemplate.ts:16]: When I ran the tests I got an error "##vso[task.issue type=error;source=TaskInternal;]Unhandled: The "path" argument must be of type string. Received undefined".
So to resolve this error, I had to add this in Tests.

Comment (z00sts) [reply Tasks/PackerBuildV1/Tests/L0CustomManagedTemplate.ts:16]: Can you please share the command you use to run the tests? It's just for me, for future usage.

**Review by z00sts — APPROVED** [review]
LGTM

**Pull request merged**